src/pymordemos/thermalblock.py

```diff
@@ -13,43 +13,47 @@
 Arguments:
   XBLOCKS    Number of blocks in x direction.
   YBLOCKS    Number of blocks in y direction.
+
   SNAPSHOTS  naive:           ignored
+
              greedy/pod:      Number of training_set parameters per block
                               (in total SNAPSHOTS^(XBLOCKS * YBLOCKS)
                               parameters).
+
              adaptive_greedy: size of validation set.
+
   RBSIZE     Size of the reduced basis
 
 Options:
   --adaptive-greedy-rho=RHO       See pymor.algorithms.adaptivegreedy [default: 1.1].
   --adaptive-greedy-gamma=GAMMA   See pymor.algorithms.adaptivegreedy [default: 0.2].
   --adaptive-greedy-theta=THETA   See pymor.algorithms.adaptivegreedy [default: 0.]
-  --alg=ALG                       The model reduction algorithm to use
+  --alg=ALG                       The model reduction algorithm to use \
```

Comment:
That causes the docstring to be continued without newline messing up the help output of the demo. Maybe we can find another way to solve this?

Reply:
Uhm, yes, we can: https://github.com/pymor/pymor/pull/866/files#diff-6f3a39e1977b1f1181f956b638289f71
Apparently I fubared applying the good solution to all the files equally.

Reply:
Actually, it turns out that sphinx is A-OK with the indents in the "options" sections and I was just too proactive in changing them